README.md

```diff
@@ -4,6 +4,59 @@
 
 This application connects tags from a Flexy to an OSIsoft PI dataserver.
 
+## Getting Started
+
+### Configuration 
+
+All user configuration is done through the ConnectorConfig.json file.  This file must be modified to match your configuration.
+
+#### Example ConnectorConfig.json
+```
+{
+   "ServerConfig":{
+      "IP":"192.168.0.124",
+      "WebID":"s0U1IjG6kMOEW7mxyHCuX2mAUEktU0VSVkVSLVBD",
+      "Credentials":"UEktU2VydmVyOk1hbmNoZXN0ZXIxMjMh"
+   },
+   "eWONConfig":{
+      "CertificatePath":"/usr/Certificates"
+   },
+   "AppConfig":{
+      "CycleTimeMs":1000
+   },
+   "TagList":["ExampleTag1", "ExampleTag2", "ExampleTag3", "ExampleTag4"]
+}
+```
+#### ServerConfig
+
+IP - IP address of your OSIsoft PI server
+
+WebID - WebID of your OSIsoft PI dataserver
+
+#### Credentials - Base64 encoded user credentials for basic authentication 
+
+To generate your Base64 encoded user credentials visit https://www.base64encode.org/ and encode "username:password"
+
+#### eWONConfig
+
+CertificatePath - Path to the directory containing your server's certificate.  For more information see the Certificates section
+
+#### AppConfig
+
+CycleTimeMs - Cycle time of the application.  All tags will be posted at this specified interval
+
+#### TagList
+
+TagList - List of eWON tags that should be connected to the OSIsoft PI server.  If PI Points with exact matching names do not exist on the PI server they will be created automatically. 
```

Comment:
Are PI points case sensitive? That could be helpful to mention here.

Reply:
They aren't actually.  I have added a note of this.